### PR TITLE
Update Calendar/Contacts Sync Tools page to cardv2

### PR DESCRIPTION
**Pull request by nitrohorse**
## Description

Resolves: #none 

#### Check List 

- [x] I have read and understand [the contributing guidelines](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- [x] I have [listed the source code](https://github.com/privacytoolsIO/privacytools.io/blob/master/source_code.md).

- ~~[ ] This project is [free/libre software](https://www.wikipedia.org/wiki/Free_software).~~

- ~~[ ] This project has an [associated discussion](https://github.com/privacytoolsIO/privacytools.io/issues).~~

* Netlify preview for the mainly edited page: https://deploy-preview-1190--privacytools-io.netlify.com/software/calendar-contacts/

* Code Repository (if applicable): N/A

### CODE REVIEW

File: _includes/sections/calendar-contacts-sync.html [hunk 1]
```diff
@@ -4,34 +4,41 @@ <h1 id="calendar_contacts" class="anchor"><a href="#calendar_contacts"><i class=
...
   image="/assets/img/provider/Nextcloud.png"
-  url="https://nextcloud.com/"
-  footer="Client OS: Windows, macOS, Linux, BSD, Unix, iOS, Android, Fire OS. Server: Linux."
-  description="NextCloud is a suite of client-server software for creating and using file hosting services. This includes calendar sync via CalDAV and contacts sync via CardDAV. Nextcloud is free and open-source, thereby allowing anyone to install and operate it without charge on a private server."
-  %}
-
-  {% include card.html color="primary"
+  description="<strong>Nextcloud</strong> is a suite of client-server software for creating and using file hosting services. This includes calendar sync via CalDAV and contacts sync via CardDAV. Nextcloud is free and open-source, thereby allowing anyone to install and operate it without charge on a private server."
+  website="https://nextcloud.com/"
+  forum=""
```
Comment (nitrohorse) [inline _includes/sections/calendar-contacts-sync.html]: TODO

File: _includes/sections/calendar-contacts-sync.html [hunk 2]
```diff
@@ -4,34 +4,41 @@ <h1 id="calendar_contacts" class="anchor"><a href="#calendar_contacts"><i class=
...
   image="/assets/img/provider/etesync.png"
-  url="https://www.etesync.com/"
-  footer="Client OS: Android, Web. Server OS: Linux."
-  description="EteSync is a secure, end-to-end encrypted and journaled personal information (e.g. contacts and calendar) cloud synchronization and backup for Android and any OS that supports CalDAV/CardDAV. It costs $14 per year to use, or you can host the server yourself for free."
-  %}
-
-</div>
+  description="<strong>EteSync</strong> is a secure, end-to-end encrypted and journaled personal information (e.g. contacts and calendar) cloud synchronization and backup for Android and any OS that supports CalDAV/CardDAV. It costs $14 per year to use, or you can host the server yourself for free."
+  website="https://www.etesync.com/"
+  forum=""
```
Comment (nitrohorse) [inline _includes/sections/calendar-contacts-sync.html]: TODO

File: _includes/sections/calendar-contacts-sync.html (hunk 1 above)
Comment (blacklight447-ptio) [reply _includes/sections/calendar-contacts-sync.html]: https://forum.privacytools.io/t/discussion-nextcloud-calender-contacts-sync-tools/1535

File: _includes/sections/calendar-contacts-sync.html (hunk 2 above)
Comment (blacklight447-ptio) [reply _includes/sections/calendar-contacts-sync.html]: https://forum.privacytools.io/t/discussion-etesync-calender-contacts-sync-tools/1536

File: _includes/sections/calendar-contacts-sync.html [hunk 3]
```diff
@@ -41,6 +48,6 @@ <h3>Worth Mentioning</h3>
   </li>
 
   <li>
-    <a href="../cloud">cloud backups</a> - consider regularly exporting your calendar and or contacts and backing them up on a separate storage drive or uploading them to cloud storage (ideally after <a href="../encryption-tools/#encrypt">encrypting</a> them).
+    <a href="../cloud">cloud backups</a> - consider regularly exporting your calendar and or contacts and backing them up on a separate storage drive or uploading them to cloud storage (ideally after <a href="../encryption-tools/">encrypting</a> them).
```
Comment (Mikaela) [inline _includes/sections/calendar-contacts-sync.html]: I think this is a bit unclear, as it's not a software unlike everything else in this section generally.

**Pull request merged**